wpeverest/everest-forms
# Fix - CSV export broken due to uppercase alphanumric meta-key

### All Submissions:






### Changes proposed in this Pull Request:

This PR aims to resolve CSV export with an empty form fields data which was occurred due to the use of uppercase alphanumeric meta-key. Additionally, like `sanitize_key` accepts dashes too.

### How to test the changes in this Pull Request:

1. Create a form with Uppercase character in field meta-key.
2. Do submit the form and export the entries CSV. You will see blank row data.
3. Check out this branch and perform a similar test, also check if dashes are accepted.

### Types of changes:




### Other information:




### Changelog entry

> Fix - CSV export is broken when meta-key is with uppercase alphanumeric value.
